lib/generators/cypress_on_rails/install_generator.rb

```diff
@@ -57,12 +57,16 @@ def add_initial_files
     end
 
     def update_files
+      if options.install_cypress
       append_to_file "#{options.cypress_folder}/support/index.js",
                      "\nimport './on-rails'",
                      after: 'import \'./commands\''
```

Comment:
@grantspeelman is this line even necessary considering the [code that it wants to append](https://github.com/shakacode/cypress-on-rails/blob/master/lib/generators/cypress_on_rails/install_generator.rb#L61) appears to now be [a default part of the file](https://github.com/shakacode/cypress-on-rails/blob/master/lib/generators/cypress_on_rails/templates/spec/cypress/support/index.js.erb#L19)